pkg/codec/vaapi/helper.c

```diff
@@ -0,0 +1,50 @@
+#include <fcntl.h>
+#include <string.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <va/va.h>
+#include <va/va_drm.h>
+
+#include "helper.h"
+
+int open2(char *path, int flags)
+{
+  return open(path, flags);
+}
```

Comment:
Why do we need to wrap `open`?

Reply:
C's `open()` has two versions and implemented as variable length argument. It's not called directly fron Go.
```c
int open(const char *pathname, int flags);
int open(const char *pathname, int flags, mode_t mode);
```

Reply:
Ah, I didn't know that Go can't call C functions that have variable length argument.